### DISCUSSION TITLE
Bosean FS-5000: Can't flash

### DISCUSSION THREAD

**Discussion by Noahtheg** [Gissio/radpro#122] [Q&A] [answered]
Hi the St-Link is new to me. I spent a couple of hours today trying to install RadPro on the Bosean FS-5000 
![Screenshot 2024-07-09 160842](https://github.com/Gissio/radpro/assets/100069792/7d9a028d-aa61-475d-a47f-e76e068d5204)
but it keeps showing the same error. I tried reconnecting everything and soldered some headers on the PCB, but nothing worked. Now I'm stuck.

**Comment by alfmck** [comment, marked as answer]
I think you've disconnected the battery, then connected ST-LINK to FS-5000 and then ST-LINK to PC USB. After that you have to see FS-5000 in measurement state. If so, +3V and ground pins are connected OK. You have to check if CLK and DIO are right connected - see also pin names on the ST-LINK board. If everything is OK, try to reinstall ST-LINK driver from original ST site or use another ST-LINK device.

**Reply by Noahtheg** [reply]
Thanks so much, I uninstalled and reinstalled it and found that one pin wasn't connected in the right place because I looked at the diagram wrong.
